src/main.rs

```diff
@@ -129,6 +129,15 @@ impl Debugger {
 
 fn main() {
     let args: Vec<String> = env::args().collect();
+
+    // mostly redundant
+    assert_eq!(args.len() > 0, true);
```

Comment:
How could this ever happen? I thought the first argument is always the name of the program? So there shall always be one argument?

Reply:
XD yah, idk why I did this tbh

Reply:
you can remove it.

Reply:
from what I know its not possible to get argc 0

Reply:
I was wrong https://stackoverflow.com/questions/49817316/can-argc-be-zero-on-a-posix-system

Reply:
That doesn't seem like a practical case to be concerned about, but I also didn't know that!